### PR TITLE
[Merged by Bors] - feat(Measure/Stieltjes): Add instance `Module ℝ≥0 StieltjesFunction`

### PR DESCRIPTION
**Pull request by LorenzoLuccioli**
- Add `StieltjesFunction.const` and `StieltjesFunction.add`
- Add lemmas `const_apply` and `add_apply`
- Add instances `AddCommMonoid StieltjesFunction` and `Module ℝ≥0 StieltjesFunction`
- Add lemmas `measure_const`, `measure_add` and `measure_smul`
- Cleanup

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Measure/Stieltjes.lean [hunk 1]
```diff
@@ -108,6 +108,57 @@ instance instInhabited : Inhabited StieltjesFunction :=
...
+/-- Constant functions are Stieltjes function. -/
+protected def const (c : ℝ) : StieltjesFunction where
+  toFun := fun _ ↦ c
+  mono' _ _ := by simp
+  right_continuous' _ := continuousWithinAt_const
+
+@[simp] lemma const_apply (c x : ℝ) : (StieltjesFunction.const c) x = c := rfl
+
+/-- The sum of two Stieltjes functions is a Stieltjes function. -/
+def add (f g : StieltjesFunction) : StieltjesFunction where
```
Comment (sgouezel) [inline Mathlib/MeasureTheory/Measure/Stieltjes.lean:120]:
```suggestion
protected def add (f g : StieltjesFunction) : StieltjesFunction where
```

**Review by sgouezel — COMMENTED** [review]
This looks great, thanks!
bors d+

**Comment by LorenzoLuccioli** [comment]
Sorry, I slightly changed the instance, separating the `AddZeroClass` instance to shorten the proof. I think it needs another quick review before merging.

**Comment by RemyDegenne** [comment]
maintainer merge

**Comment by sgouezel** [comment]
bors r+
Thanks!

**Pull request closed without merging**